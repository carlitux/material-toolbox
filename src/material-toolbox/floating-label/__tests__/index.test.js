// @flow
import React from 'react';
import { shallow } from 'enzyme';
import FloatingLabel from '../index';

describe('component::FloatingLabel', () => {
  it('Should match snapshoot', () => {
    const component = shallow(<FloatingLabel label="test" />);
    expect(component).toMatchSnapshot();
  });

  it('Should match snapshoot cssOnly', () => {
    const component = shallow(<FloatingLabel label="test" cssOnly />);
    expect(component).toMatchSnapshot();
  });

  it('Should match snapshoot with preFilled', () => {
    const component = shallow(<FloatingLabel label="test" preFilled />);
    expect(component).toMatchSnapshot();
  });
});
