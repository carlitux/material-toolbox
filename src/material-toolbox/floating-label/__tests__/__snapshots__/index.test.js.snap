// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component::FloatingLabel Should match snapshoot 1`] = `
<span
  className="mdc-floating-label"
>
  test
</span>
`;

exports[`component::FloatingLabel Should match snapshoot cssOnly 1`] = `
<span
  className="mdc-floating-label"
>
  test
</span>
`;

exports[`component::FloatingLabel Should match snapshoot with preFilled 1`] = `
<span
  className="mdc-floating-label mdc-floating-label--float-above"
>
  test
</span>
`;
